src/SFA.DAS.Commitments.Database/PostDeployment/PostDeploymentScript.sql

```diff
@@ -10,3 +10,38 @@ Post-Deployment Script Template
 --------------------------------------------------------------------------------------
 */
 
+IF EXISTS (SELECT COUNT(*) FROM [dbo].[Apprenticeship] WHERE [PaymentStatus] = 2 AND [PauseDate] = null)
```

Comment:
**IS NULL**  instead of **= null**?
Does **= null** work for you?

Reply:
Good comment, right question, but it did work when I tested it. :-D 
I will put it right though.

Reply:
Tested it again and it had another problem COUNT(*) changed to  * and then 'IS NULL' or  '= NULL' works either way. Have left it with IS NULL.